boxes/identity-server/provisioner/provisioner.sh

```diff
@@ -60,12 +66,20 @@ if test -d ${JAVA_HOME}; then
 fi
 echo "Successfully set up Java"
 
-# unpack the WSO2 product pack to the working directory
-echo "Setting up the ${WSO2_SERVER}-${WSO2_SERVER_VERSION} server..."
+# set up WUM
+echo "Setting up WUM..."
+if test ! -d ${WUM_HOME}; then mkdir ${WUM_HOME}; fi
+if test -d ${WUM_HOME}; then
+  tar -xzf ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} -C ${WUM_HOME} --strip-components=1
```

Comment:
Please refer the above review regarding the usage of `z` argument in `tar` command.

Reply:
fixed